examples/fscloud/variables.tf

```diff
@@ -69,3 +69,15 @@ variable "allow_at_to_cos" {
   description = "Set rule for Activity Tracker to COS, default is true"
   default     = true
 }
+
+variable "kms" {
+  type        = list(string)
+  description = "List specific endpoint types for target services, valid values for endpoints are 'public', 'private' or 'direct'"
```

Comment:
Seems to be irrelevant here - copy pasted from somewhere else?

Reply:
Yes, my bad, resolved now.